lib/puppet/type/sensu_ad_auth.rb

```diff
@@ -57,7 +76,7 @@
     Keys:
     * host: required
     * port: required
-    * group_search: required
+    * group_search: optional (omit to use memberOf)
```

Comment:
A note here that says it is required with Sensu 5 and as of Sensu 6 it is optional.

Reply:
Note updated.